keps/sig-node/3857-rro-mounts/README.md

```diff
@@ -631,9 +638,23 @@ We expect no non-infra related flakes in the last month as a GA graduation crite
 - run RecursiveReadOnly="Enabled", and verify that the mount is actually recursively read-only
 - run RecursiveReadOnly="Disabled", and verify that the mount is actually not recursively read-only
 
-Tests are implemented in <https://github.com/kubernetes/kubernetes/blob/v1.30.0/test/e2e_node/mount_rro_linux_test.go>,
-and will be executed on the CI when the CI is upgraded to use containerd v2.0.
-So, there is no link to the testgrid yet.
+The `e2e_node` tests are implemented in <https://github.com/kubernetes/kubernetes/blob/v1.30.0/test/e2e_node/mount_rro_linux_test.go>.
+
+Test grid:
+
+- [`containerd-node-e2e-features-1.7`](https://testgrid.k8s.io/sig-node-containerd#containerd-node-e2e-features-1.7)
+```
+E2eNode Suite.[It] [sig-node] Mount recursive read-only [LinuxOnly] [Feature:RecursiveReadOnlyMounts] Mount recursive read-only when the runtime does not support recursive read-only mounts should accept non-recursive read-only mounts
+E2eNode Suite.[It] [sig-node] Mount recursive read-only [LinuxOnly] [Feature:RecursiveReadOnlyMounts] Mount recursive read-only when the runtime does not support recursive read-only mounts should reject recursive read-only mounts
+```
+
+- [`ci-crio-cgroupv2-node-e2e-features`](https://testgrid.k8s.io/sig-node-cri-o#ci-crio-cgroupv2-node-e2e-features)
+```
+E2eNode Suite.[It] [sig-node] Mount recursive read-only [LinuxOnly] [Feature:RecursiveReadOnlyMounts] Mount recursive read-only when the runtime supports recursive read-only mounts should accept recursive read-only mounts
+E2eNode Suite.[It] [sig-node] Mount recursive read-only [LinuxOnly] [Feature:RecursiveReadOnlyMounts] Mount recursive read-only when the runtime supports recursive read-only mounts should reject invalid recursive read-only mounts
+```
+
+<!-- There is no link to `k8s-triage`, as the tests haven't failed as of the time of writing this (Feb 2025). -->
```

Comment:
Have a look at the raw [e2e section](https://github.com/kubernetes/enhancements/blob/master/keps/NNNN-kep-template/README.md#e2e-tests) so link like https://storage.googleapis.com/k8s-triage/index.html?sig=node&job=e2e&test=recursive%20read-only is what's required here.

Reply:
Added